tests/dbt/test_graph.py

```diff
@@ -39,45 +54,116 @@ def test_load_via_manifest_with_exclude(pipeline_name, manifest_filepath, model_
 
 @patch("cosmos.dbt.graph.DbtGraph.load_from_dbt_manifest", return_value=None)
 def test_load_automatic_manifest_is_available(mock_load_from_dbt_manifest):
-    dbt_project = DbtProject(name="jaffle_shop", root_dir=DBT_PROJECTS_ROOT_DIR, manifest_path=SAMPLE_MANIFEST)
-    dbt_graph = DbtGraph(project=dbt_project)
-    dbt_graph.load(execution_mode=ExecutionMode.LOCAL)
+    dbt_graph = DbtGraph(
```

Comment:
It feels that we are overusing the overall config abstraction. The previous interface of `DbtGraph` was clear, and simpler to understand and test.